package.json

```diff
@@ -24,7 +24,6 @@
     "test": "web-test-runner --files test/**/*.test.ts --node-resolve --playwright --coverage --browsers chromium firefox webkit",
     "lint": "pnpm run \"/^lint:.*/\"",
     "lint:prettier": "prettier --check . --log-level warn",
-    "lint:prettier-fix": "prettier --write . --log-level warn",
```

Comment:
FYI, reverting this because (A) we have the `"format"` script which does this, and (B) running `pnpm lint` runs all `lint:*` scripts in parallel, so it could cause weird race conditions where the files are changing while the linters are running checks.